Tutorials/Editors-Manual/Working-with-Content/Rich-Text-Editor/index.md

```diff
@@ -195,15 +198,15 @@ You can upload images directly from the rich text editor on the page you are edi
 5. In the Title field, enter a name/description for the image. (By default the name of the file will be automatically entered for your convenience).
 6. Click ***Insert** to add the image.
 
-![mediaUpload.jpg](images/mediaUpload.jpg)
+![mediaUpload.jpg](images/Upload-media.png)
```

Comment:
This image - I'm not sure where it belongs?
It's an image of the search feature. Perhaps it should have been another image here? 🤔